test/services/RepositoryServiceSpec.scala

```diff
@@ -69,7 +69,7 @@ class RepositoryServiceSpec extends BaseSpecWithApplication with MockFactory {
     "RepositoryService .create()" should {
 
         "get a DataModel" in {
-            (mockRepository.create(_: DataModel)).expects(dataModel)
+            (mockRepository.create(_: DataModel)).expects(*)
                 .returning(Future(Right(dataModel))).once()
```

Comment:
Be careful using `expects(*)` - this is something you may see in existing code but is not best practice. Try to be specific about what you are expecting to be returned.